extensions/LIZARD_OFFICIAL/2FA.js

```diff
@@ -0,0 +1,37 @@
+
+  "use strict";
+
+  class MFA {
+    getInfo() {
+      return {
+        id: "twofactorauth",
```

Comment:
The extension ID should contain someway to identify it's made by you. Something like "lizard2fauth" would work.